lexery/__init__.py

```diff
@@ -190,11 +210,10 @@ def lex(self, text: str) -> List[List[Token]]:
             while position < len(line):
                 mtched = False
                 for rule in self.rules:
-                    mtch = rule.pattern.match(line, position)
+                    _line = line
```

Comment:
Please rename `_line` to `another_line`. Ditto for `_token` to `another_token`.

We use `_*` naming convention for private/protected members of a class.